cpp/proto/serializer.h

```diff
@@ -26,71 +26,25 @@ cert_trans::serialization::SerializeResult CheckSctExtensionsFormat(
 void WriteSctExtension(const repeated_sct_extension& extension,
                        std::string* output);
 
-class TLSDeserializer {
- public:
-  // We do not make a copy, so input must remain valid.
-  // TODO(pphaneuf): And so we should take a string *, not a string &
-  // (which could be to a temporary, and not valid once the
-  // constructor returns).
-  explicit TLSDeserializer(const std::string& input);
-
-  bool ReachedEnd() const {
-    return bytes_remaining_ == 0;
-  }
+namespace serialization {
```

Comment:
This should be inside of `namespace cert_trans` as well, which would remove the need for much qualifying (for example, `cert_trans::serialization::DeserializeResult` a few lines below becomes just `DeserializeResult`).